### PR TITLE
flow trigger UI

### PR DESCRIPTION
**Pull request by chengren311**
This PR adds an UI page to display current scheduled flow trigger:
image

### CODE REVIEW

File: azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java [hunk 1]
```diff
@@ -508,6 +509,7 @@ private void configureRoutes() throws TriggerManagerException {
     root.addServlet(new ServletHolder(new StatusServlet(this.statusService)), "/status");
     root.addServlet(new ServletHolder(new NoteServlet()), "/notes");
     root.addServlet(new ServletHolder(new FlowTriggerInstanceServlet()), "/flowtriggerinstance");
+    root.addServlet(new ServletHolder(new FlowTriggerServlet()), "/flowtrigger");
```
Comment (kunkun-tang) [inline azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java]: Is there a button for click to jump to the page?

File: azkaban-web-server/src/main/java/azkaban/webapp/servlet/FlowTriggerServlet.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
```
Comment (kunkun-tang) [inline azkaban-web-server/src/main/java/azkaban/webapp/servlet/FlowTriggerServlet.java]: 2018

File: azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    var errorMessage = null;
+    var successMessage = null;
+
+    $(document).ready(function () {
+      var jobTable = $("#executingJobs");
+      jobTable.tablesorter();
+    });
+  </script>
+  <script>
+    function showTriggerProperties(flowTrigger) {
```
Comment (kunkun-tang) [inline azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm]: Never used?

File: azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm [hunk 4]
```diff
@@ -0,0 +1,173 @@
+#*
+ * Copyright 2012 LinkedIn Corp.
```
Comment (kunkun-tang) [inline azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm]: 2018

File: azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java (hunk 1 above)
Comment (chengren311) [reply azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java]: currently no, but will add it in future.

File: azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm (hunk 3 above)
Comment (chengren311) [reply azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm]: removed

File: azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm (hunk 4 above)
Comment (chengren311) [reply azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm]: updated

File: azkaban-web-server/src/main/java/azkaban/webapp/servlet/FlowTriggerServlet.java (hunk 2 above)
Comment (chengren311) [reply azkaban-web-server/src/main/java/azkaban/webapp/servlet/FlowTriggerServlet.java]: updated

**Pull request merged**